test/EndToEnd/ProjectTemplates/NetCoreConsoleApp.zip/NetCoreConsoleApp.vstemplate

```diff
@@ -14,8 +14,8 @@
     <Icon>__TemplateIcon.ico</Icon>
   </TemplateData>
   <TemplateContent>
-    <Project TargetFileName="ConsoleApp.csproj" File="ConsoleApp.csproj" ReplaceParameters="true">
-      <ProjectItem ReplaceParameters="true" TargetFileName="Program.cs">Program.cs</ProjectItem>
+    <Project File="ConsoleApp.csproj" ReplaceParameters="true">
+      <ProjectItem ReplaceParameters="true" OpenInEditor="false">Program.cs</ProjectItem>
```

Comment:
The idea was to optimize templates for E2E tests automation to avoid opening editor. `OpenInEditor` is `false` by default in some cases but for some templates I had to specify it explicitly.